### ISSUE TITLE
Connection Timeout when connecting to NordVPN

### ISSUE THREAD

**Issue by jeffmartinxyz** [haugene/docker-transmission-openvpn#672]
Looks like something similiar to #492 may be happening with NordVPN. Started seeing these log entries:

```
Using OpenVPN provider: NORDVPN
Setting best server us3089.nordvpn.com.udp
Supplied config us3089.nordvpn.com.udp.ovpn could not be found.
Using default OpenVPN gateway for provider nordvpn
Setting OPENVPN credentials...
adding route to local network 10.0.0.0/8 via 10.0.0.1 dev enp0s25
Sat Dec 22 21:33:06 2018 OpenVPN 2.4.6 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Apr 24 2018
Sat Dec 22 21:33:06 2018 library versions: OpenSSL 1.0.2g  1 Mar 2016, LZO 2.08
Sat Dec 22 21:33:06 2018 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Sat Dec 22 21:33:06 2018 NOTE: --fast-io is disabled since we are not using UDP
Sat Dec 22 21:33:06 2018 Outgoing Control Channel Authentication: Using 160 bit message hash 'SHA1' for HMAC authentication
Sat Dec 22 21:33:06 2018 Incoming Control Channel Authentication: Using 160 bit message hash 'SHA1' for HMAC authentication
Sat Dec 22 21:33:06 2018 TCP/UDP: Preserving recently used remote address: [AF_INET]185.30.232.17:443
Sat Dec 22 21:33:06 2018 Socket Buffers: R=[87380->87380] S=[16384->16384]
Sat Dec 22 21:33:06 2018 Attempting to establish TCP connection with [AF_INET]185.30.232.17:443 [nonblock]
Sat Dec 22 21:35:06 2018 TCP: connect to [AF_INET]185.30.232.17:443 failed: Connection timed out
Sat Dec 22 21:35:06 2018 SIGUSR1[connection failed(soft),init_instance] received, process restarting
Sat Dec 22 21:35:06 2018 Restart pause, 5 second(s)
Sat Dec 22 21:35:11 2018 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Sat Dec 22 21:35:11 2018 NOTE: --fast-io is disabled since we are not using UDP
```

Are the .OVPN files out of date?

**Comment by jeffmartinxyz** [comment]
Works again today. Not sure what the problem was.

**Comment by haugene** [comment]
NordVPN has so many OpenVPN files, and they seem to change quite frequent. The config files are updated 8 days ago, but still it look like the "find the best server" script picked one that wasn't in the container.

Could be fixed by something like this suggested in #655 
Let's see where everything goes, but can we close this for now and follow up in #655?

**Comment by jeffmartinxyz** [comment]
Sounds like a good way forward, thanks for making this!!!
